src/scripts/main.js

```diff
@@ -358,3 +358,35 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+people.forEach(person => {
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+  nameCell.textContent = person.name;
+  row.appendChild(nameCell);
+
+  const genderCell = document.createElement('td');
+  genderCell.textContent = person.sex;
+  row.appendChild(genderCell);
+
+  const bornCell = document.createElement('td');
+  bornCell.textContent = person.born;
+  row.appendChild(bornCell);
+
+  const diedCell = document.createElement('td');
+  diedCell.textContent = person.died;
+  row.appendChild(diedCell);
+
+  const ageCell = document.createElement('td');
+  ageCell.textContent = age;
+  row.appendChild(ageCell);
+
+  const centuryCell = document.createElement('td');
+  centuryCell.textContent = century;
+  row.appendChild(centuryCell);
+
+  document.querySelector('.dashboard tbody').appendChild(row);
+});
```

Comment:
Ensure that the HTML document contains a table with a class of 'dashboard' and a 'tbody' element. Without this structure, the code will not append the rows correctly, and the data will not be displayed.